### ISSUE TITLE
Add support for WAI-ARIA aria-live property for portions of the page updated via <f:ajax>

### ISSUE THREAD

**Issue by gurunrao** [gurunrao/issue_migration#1159]
Currently, JSF doesn't have any explicit support for WAI-ARIA ([http://www.w3.org/WAI/intro/aria.php](http://www.w3.org/WAI/intro/aria.php)), except for the role attribute. <f:ajax> should be updated to use the aria-live attribute at the very least.

See EG thread here: [http://java.net/projects/javaserverfaces-spec-public/lists/jsr344-experts/archive/2013-01/message/5](http://java.net/projects/javaserverfaces-spec-public/lists/jsr344-experts/archive/2013-01/message/5)

**Comment by gurunrao** [comment]
**Closing this as this issue is migrated to** https://github.com/gurunrao/issue_migration3/issues/1159
